TTTSwiftTests/TTTAcceptanceTests.swift

```diff
@@ -0,0 +1,19 @@
+import Quick
+import Nimble
+import TTTSwift
+
+class TTTAcceptanceTests: QuickSpec {
+    override func spec() {
+        
+        describe("a full game of tic-tac-toe between two first available space computers plays through and exits properly") {
+            
+            it("plays the game") {
+                let game = Game(playerOne: FirstAvailableSpotComputerPlayer(), playerTwo: FirstAvailableSpotComputerPlayer())
+                game.play()
+                expect(game.isInProgress()).to(beFalse())
```

Comment:
This test could use some more assertions. The setup and action are good, however it might be valuable to inspect what the board looks like.